mlflow/genai/judges/__init__.py

```diff
@@ -8,6 +9,7 @@
 )
 
 __all__ = [
+    "CategoricalRating",
```

Comment:
@smoorjani Do judges called from the MLflow namespace return instances of this MLflow-namespaced entity, or do they still return instances of the databricks-agents-namespaced entity?

The objective is to ensure that MLflow-namespaced judges return MLflow-namespaced entities.

Reply:
Updated!